test/e2e/create_test.go

```diff
@@ -58,6 +58,22 @@ func testWindowsNodeCreation(t *testing.T) {
 	// Create a private key secret with the known private key.
 	require.NoError(t, testCtx.createPrivateKeySecret(true), "could not create known private key secret")
 
+	//  Machine API Operator is not available in a platform=none UPI cluster. Hence, cannot use machineSets to create
+	//  Windows instances.
+	//  Requirements for platform=none:
+	//  	a) the Windows instance(s) must be provisioned, and
+	//  	b) the windows-instances configMap must exist pointing to the given Windows instance(s)
+	if testCtx.CloudProvider.GetType() == config.NonePlatformType {
+		// TODO: setPowerShellDefaultShell for given Windows instance(s)
+		// TODO: validate windows-instances configMap
+		t.Run("Windows instance configured by ConfigMap controller", func(t *testing.T) {
```

Comment:
All BYOH nodes will be configured by configmap controller, how about a name that suggests this tests the BYOH configuration on platform none provider?

Reply:
Sure, will include more context around "platform none provider".